automation/terraform/modules/kubernetes/testnet/variables.tf

```diff
@@ -257,31 +252,43 @@ variable "make_report_accounts" {
   default = ""
 }
 
-# Archive-Postgres Vars
-
-variable "archive_persistence_enabled" {
-  type    = bool
-  default = true
-}
+# Archive | Postgres Vars
 
-variable "archive_persistence_class" {
-  type    = string
-  default = "ssd"
-}
-
-variable "archive_persistence_reclaim_policy" {
-  type    = string
-  default = "retain"
-}
-
-variable "archive_persistence_access_modes" {
-  type    = list
-  default = ["ReadWriteOnce"]
+variable "archive_configs" {
+  type = list(
+    object({
+      name               = string
+      image              = optional(string)
+      serverPort         = optional(string)
+      externalPort       = optional(string)
+      postgresHost       = optional(string)
+      postgresPort       = optional(string)
+      postgresDB         = optional(string)
+      postgresqlUsername = optional(string)
+      postgresqlPassword = optional(string)
+      remoteSchemaFile   = optional(string)
+      enableLocalDaemon  = optional(bool)
+      enablePostgresDB   = optional(bool)
+    })
+  )
+  default = []
 }
 
-variable "archive_persistence_size" {
-  type    = string
-  default = "8Gi"
+variable "persistence_config" {
+  type = object({
+      enabled            = optional(bool)
+      size               = optional(string)
+      reclaimPolicy      = optional(string)
+      storageClass       = optional(string)
+      accessModes       = optional(list(string))
+  })
+  default = {
+    enabled = true
+    size  = "8Gi"
+    reclaimPolicy = "retain"
+    storageClass  = "us-west1-ssd-retain"
```

Comment:
rename storage class to remove redundant ${region} component e.g: us-west-1-ssd-retain => ssd-retain